chroma_core/models/host.py

```diff
@@ -1683,20 +1684,17 @@ def get_steps(self):
             target = target.downcast()
             primary_tm = target.managedtargetmount_set.get(primary=True)
             steps.append((MountOrImportStep, MountOrImportStep.create_parameters(target, primary_tm.host, False)))
```

Comment:
Is this step needed?  Ensuring ZPOOLs were imported was important for writeconf, but not for replace nids.

Reply:
Seems to work fine w/o it, removing